ctru-rs/src/error.rs

```diff
@@ -17,12 +19,22 @@ impl From<i32> for Error {
 impl fmt::Debug for Error {
```

Comment:
I can't put a comment on the right line, but we could replace all `i32` in this file with `ctru_sys::Result`. Not necessary though, but would be clearer.